monkey/infection_monkey/monkey.py

```diff
@@ -72,26 +72,23 @@ def initialize(self):
         arg_parser = argparse.ArgumentParser()
         arg_parser.add_argument('-p', '--parent')
         arg_parser.add_argument('-t', '--tunnel')
-        arg_parser.add_argument('-s', '--server')
+        arg_parser.add_argument('-s', '--servers', nargs = '+')
         arg_parser.add_argument('-d', '--depth', type=int)
         self._opts, self._args = arg_parser.parse_known_args(self._args)
 
         self._parent = self._opts.parent
         self._default_tunnel = self._opts.tunnel
-        self._default_server = self._opts.server
+        self._default_servers = self._opts.servers
 
         if self._opts.depth:
             WormConfiguration._depth_from_commandline = True
         self._keep_running = True
         self._network = NetworkScanner()
         self._dropper_path = sys.argv[0]
-
-        if self._default_server:
-            if self._default_server not in WormConfiguration.command_servers:
-                LOG.debug("Added default server: %s" % self._default_server)
-                WormConfiguration.command_servers.insert(0, self._default_server)
-            else:
-                LOG.debug("Default server: %s is already in command servers list" % self._default_server)
+        
+        def_server_list = [ def_server for def_server in WormConfiguration.command_servers ]
+        WormConfiguration.command_servers = [ server for server in self._default_servers ]
```

Comment:
Why do you use list comprehension instead of simple assignment. what's the difference between `def_server_list = [ def_server for def_server in WormConfiguration.command_servers ]` and `def_server_list = WormConfiguration.command_servers`?

Reply:
Sry, Had a silly doubt,Corrected